probe-rs/Cargo.toml

```diff
@@ -106,6 +106,7 @@ clap = { version = "4", features = ["derive"] }
 itm = { version = "0.9.0-rc.1", default-features = false }
 pretty_assertions = "1"
 test-case = "3"
+test-log = { version = "0.2.16", features = [ "trace" ] }
```

Comment:
Do you use this anywhere?

Reply:
ah good catch, it should be in `adi_v5_memory_interface::test` for now and could be anywhere else.